src/compiler/checker.ts

```diff
@@ -90,6 +90,9 @@ module ts {
         let unknownType = createIntrinsicType(TypeFlags.Any, "unknown");
 
         let emptyObjectType = createAnonymousType(undefined, emptySymbols, emptyArray, emptyArray, undefined, undefined);
+        let emptyGenericType = <GenericType><ObjectType>createAnonymousType(undefined, emptySymbols, emptyArray, emptyArray, undefined, undefined);

```

Comment:
Okay, I can do that. I will still have to cast it though, in order to use it as a generic type.